tests/e2e/page_objects/dashBoard.ts

```diff
@@ -32,19 +33,27 @@ const dashBoardPage = {
     menuText: {
       selector: "div.v-subheader.font-weight-bold.text-h6.theme--light",
     },
-
+    getStartedBtn: {
+      selector:
+        "button.ml-auto.v-btn.v-btn--is-elevated.v-btn--has-bg.theme--light.v-size--default.primary",
+    },
     dashboardTab: {
       selector: "a[href*='createportfolio']",
     },
     myPortfoliosTab: {
       selector: "a[href*='portfolios']",
     },
     reportsTab: {
-      selector: "a[href*='dashboard#']",
+      selector: "a[href*='createportfolio#']",
```

Comment:
There should only be one object that has the link to `createportfolios`, please remove the `*`

Reply:
Tests are  failing  without it. So definitely need * there.